backend/app/app.go

```diff
@@ -125,5 +127,19 @@ func Start(
 		panic(err)
 	}
 
-	httpAPI.StartAndWait(config.HTTPAPIPort)
+	httpAPI.StartAsync(config.HTTPAPIPort)
+
+	gRPCService, err := dep.InjectGRPCService(
+		env.Runtime(config.Runtime),
+		provider.LogPrefix(config.LogPrefix),
+		config.LogLevel,
+		sqlDB,
+		security.Policy{},
```

Comment:
This is empty?! I think we need to provide the path of certificate & secret key. And enable TLS to encrypt traffic between the API consumer and our backend.

Reply:
Please remember to update GitOps before deploying: https://github.com/short-d/gitops/blob/master/apps/kgs/production.yaml#L71-L78